WalletLibrary/WalletLibrary/Requests/Manifest/ContractIssuanceRequest.swift

```diff
@@ -53,14 +64,58 @@ class ContractIssuanceRequest: VerifiedIdIssuanceRequest {
         do {
             try self.responseContainer.add(requirement: requirement)
             let verifiedId = try await verifiedIdRequester.send(request: responseContainer)
+            
+            await sendResultIfCallbackAndStateExist(wasIssuanceSuccessful: true)
+            
             return Result.success(verifiedId)
         } catch {
+            await sendResultIfCallbackAndStateExist(wasIssuanceSuccessful: false,
+                                                    withErrorDetails: .issuanceServiceError)
             return Result.failure(error)
```

Comment:
Is it the only error supported during issuance completion on iOS? We would also want to add it for fetching request failing scenario.

Reply:
done.